README.md

```diff
@@ -338,6 +338,18 @@ function Profile () {
 }
 ```
 
+In the example above, when clicking the button, it will send the request, **then** update the client data and 
```

Comment:
I think this is a little confusing because we're referring to two different requests here, a POST request to change the data in your remote data source, and then another GET request (the revalidation).

Here's my attempt to clarify the steps:

```suggestion
Clicking the button in the example above will send a POST request to modify the remote data, locally update the client data and
```

Reply:
Thanks! That’s indeed clearer to me 👏